composer.json

```diff
@@ -34,6 +34,7 @@
 		"test": "phpunit"
 	},
 	"require-dev": {
-		"scrutinizer/ocular": "^1.3"
+		"scrutinizer/ocular": "^1.3",
+		"phpunit/phpunit": "^5.2"
```

Comment:
Note that phpunit 5 requires PHP 5.6 as a minimum, did we decide on our minimum supported PHP version already? It was a question in the Roadmap document: https://github.com/phpab/phpab/wiki/Roadmap